### DISCUSSION TITLE
Load testing results

### DISCUSSION THREAD

**Discussion by rjcampion3** [bluenviron/mediamtx#1643] [Questions]
Do you have any load testing results available? Total number of streams, clients or anything like that? Just want to understand horizontal scaling and if multiple rtsp instances need to run concurrently. 

Thanks

**Comment by aler9** [comment]
Hello, test load results entirely depends on the target hardware. Just launch an instance on a single machine and measure how many clients you can connect to. Personally i can say that the two main bottlenecks are usually

* the network bandwidth
* the maximum number of TCP/UDP clients per IP address, which is 2^16 = 65536

Machine capabilities (CPU / RAM) are not much relevant.

I've run instances with 2000 streams each and thousands of clients.

[a comment by an automated account is not shown]
